dd-trace-ot/src/main/java/datadog/opentracing/PendingTrace.java

```diff
@@ -228,6 +229,13 @@ private void expireReference() {
   private synchronized void write() {
     if (isWritten.compareAndSet(false, true)) {
       removePendingTrace();
+      if (size() > Config.get().getMaxSpansPerTrace()) {
+        log.error(
+            "Not writing trace of {} spans, exceeds configured max of {}",
```

Comment:
Does "not writing" mean dropping? It is not quite clear. Would it make sense to rephrase?

Reply:
Good idea; I will change the message.